data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -310,6 +388,25 @@ class ApiServerClient {
       });
   }
 
+  /**
+   * @description Function to Update the error field
+   * @param {string} opts.operationName - Name of operation
+   * @param {string} opts.operationType - Type of operation
+   * @param {string} opts.operationId - Unique id of operation
+   * @param {Object} opts.error - Value to set as error
+   */
+  updateOperationError(opts) {
+    const change = {
```

Comment:
would rather rename this attrib to `response` or `operationStatus`

Reply:
Done